docs/execution-providers/STVM-ExecutionProvider.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Standalone TVM (STVM)
+description: Instructions to execute ONNX Runtime with the Standalone TVM (STVM) execution provider
+parent: Execution Providers
+nav_order: 15
+redirect_from: /docs/reference/execution-providers/STVM-ExecutionProvider
+---
+
+# Standalone TVM (STVM) Execution Provider
+{: .no_toc }
+
+STVM is an execution provider for ONNX Runtime that is built on top of Apache TVM. It enables ONNX Runtime users to leverage Apache TVM model optimizations.
+STVM EP is currently in "Preview". It's been tested to work on a handful of models on Linux, but not on Windows or MacOS.
+
+## Contents
+{: .no_toc }
+
+* TOC placeholder
+{:toc}
+
+## Build
+
+To use the STVM EP in ONNX Runtime (ORT), users first need to build Apache TVM and ONNX Runtime.
+
+Note: some python packages may need to be upgraded/downgraded because both TVM and ORT with the STVM EP use the Python API. Alternatively, use modify PYTHONPATH to solve these conflicts. 
+
+### Build and configure TVM
+
+Install the minimal pre-requisites on Ubuntu/Debian like linux operating systems:
+```bash
+apt-get install -y python3 python3-dev python3-pip python3-setuptools gcc libtinfo-dev zlib1g-dev build-essential cmake libedit-dev libxml2-dev llvm-12
+pip3 install numpy decorator attrs
+```
+
+Clone this repo using the `--recursive` flag to pull all associated dependencies
```

Comment:
We should say which repo: assume https://github.com/microsoft/onnxruntime

Reply:
That's correct we'll make the correction, thank you